edx/edx-enterprise
# [ENT-96] Add details to EnterpriseCustomerUser; link from Manage Learners

This pull request enhances the EnterpriseCustomerUser admin panel to contain the following additional information:
* Username
* email address
* Added date
* Data Sharing Consent status/history

In addition, the Manage Learners template has been enhanced to link directly to each EnterpriseCustomerUser.

**JIRA tickets**: Implements ENT-96

**Dependencies**: None

**Screenshots**:

<img width="780" alt="screen shot 2016-12-15 at 4 57 02 pm" src="https://cloud.githubusercontent.com/assets/7773758/21243626/aa2a3216-c2e7-11e6-8cbb-7181099fcf05.png">


<img width="943" alt="screen shot 2016-12-15 at 4 57 29 pm" src="https://cloud.githubusercontent.com/assets/7773758/21243633/b2bbdac4-c2e7-11e6-8405-06969ff74792.png">


**Sandbox URL**: https://ent96.opencraft.hosting/ (direct IP: http://213.32.75.54; use `staff` account for superuser access)

**Merge deadline**: Jan. 9 2016

**Testing instructions**:

1. In the sandbox, go to the Manage Learner view for the existing EnterpriseCustomer
2. Observe that the username for each linked user is now a link, rather than plain text
3. Click the link, and observe that you're taken to the EnterpriseCustomerUser admin
4. Observe that user details are visible, but not editable, on the admin.
5. Observe that historical UserDataSharingConsentAudit items are listed below the user details.

**Author notes and concerns**:


1. Note that at present, a "status" item is not shown; this is because all EnterpriseCustomerUsers are linked to EnterpriseCustomers.


**Reviewers**

**Merge checklist:**
